src/auth/basic/LDAP/basic_ldap_auth.cc

```diff
@@ -134,6 +134,10 @@ PFldap_start_tls_s Win32_ldap_start_tls_s;
 #include <lber.h>
 #include <ldap.h>
 
+#if _SQUID_SOLARIS_
```

Comment:
Should we generalize this to other "broken" platforms?

```suggestion
#ifndef LDAP_SECURITY_ERROR
```

Reply:
this is the only unix I stumbled across this but not against the change.